papers/whitepaper.md

```diff
@@ -228,6 +228,10 @@ Implementations must account for this possibility by aborting any ongoing initia
 
 In practice these delays cause participants to take turns acting as initiator and acting as responder since the ten seconds difference is usually enough for the handshake with switched roles to complete before the old initiator's rekey timer goes to zero.
 
+## Endianess
```

Comment:
nit: It is spelled "Endianness"

Reply:
Besides this, is this section even required. As far as I can tell, the protocol does not transmit any integers, but only single-byte integers or binary data.

Reply:
Feel free to use your push rights for spell correction, but do notify me please so I do not accidentally overwrite your commit.

> I can see there is a lot of ProVerif code covering secrecy and availability but does any of it cover (or overlap with) identity hiding?